OasisDEX/oasis-borrow
# [sc-12017] - RPC Gateway

# [Title](https://shortcutLink)

  
## Changes 👷‍♀️
- added `rpcGateway` API endpoint that call RPC Gateway instead of provider directly https://github.com/OasisDEX/rpc-gateway
- added `UseRpcGateway` feature flag
- added `RPC_GATEWAY` env
- added method to fetch the config and cache for backend with default TTL of 5 minutes
  
## How to test 🧪
- set `UseRpcGateway` to `true` locally
- set `RPC_GATEWAY` to https://discord.com/channels/837076147694207067/1156237535689850990/1156239716505620510
- see if it works :1st_place_medal:
